code/game/objects/structures/false_walls.dm

```diff
@@ -315,8 +293,8 @@
 	icon = 'icons/turf/walls/iron_wall.dmi'
 	icon_state = "iron_wall-0"
 	base_icon_state = "iron_wall"
-	mineral = /obj/item/stack/rods
-	mineral_amount = 5
+	mineral = /obj/item/stack/sheet/iron
+	mineral_amount = 2
```

Comment:
jesse

Reply:
what the fuck are you doing

Reply:
okay sorry

Reply:
wait this DID fix the runtime, I'll find a better solution.